tests/django_restframework_gis_tests/tests.py

```diff
@@ -489,3 +489,34 @@ def test_geojson_HTML_widget_value(self):
         # doesn't seem possible with DRF 3.0
         # let's wait for DRF 3.1
         #self.assertNotContains(response, 'u&#39;type&#39;: u&#39;Point&#39;, u&#39;coordinates&#39;:')
+
+    def test_patch_geojson_location(self):
+        location = Location.objects.create(name='geojson patch test', geometry='POINT (135.0 45.0)')
+
+        url = reverse('api_geojson_location_details', args=[location.id])
+        data = {
+            "properties": {
+                "name":"geojson successful patch test"
+            },
+            "geometry": "POINT (10.1 10.1)"
```

Comment:
testing for the acceptance of WKT strings like "POINT (10.1 10.1)" was done mainly to ensure WKT would work in order to allow flexibility, but here it would probably make more sense to have a GeoJSON Point: `{ "type": "Point", "coordinates": [10.1, 10.1] }`